packages/core/lib/commands/migrate/meta.js

```diff
@@ -43,6 +43,11 @@ module.exports = {
       describe: "Adds extra verbosity to the status of an ongoing migration",
       type: "boolean",
       default: false
+    },
+    "save": {
+      describe: "Specify whether the migration will save on chain",
+      type: "boolean",
+      default: true
```

Comment:
I think this should be good to default to `true` since that will maintain current expected behavior.